src/s11/main.c

```diff
@@ -82,17 +84,17 @@ handle_mmeapp_message_s11(void * data)
 void * tipc_msg_handler_s11()
 {
 	int bytesRead = 0;
-	unsigned char buffer[1024] = {0};
+	unsigned char buffer[S11_IPC_MSG_BUF_LEN] = {0};
 	while (1)
 	{
-		if ((bytesRead = read_tipc_msg(ipc_reader_tipc_s11, buffer, 1024)) > 0)
+		if ((bytesRead = read_tipc_msg(ipc_reader_tipc_s11, buffer, S11_IPC_MSG_BUF_LEN)) > 0)
 		{
 			unsigned char *tmpBuf = (unsigned char *) malloc(sizeof(char) * bytesRead);
 			memcpy(tmpBuf, buffer, bytesRead);
 			log_msg(LOG_INFO, "S11 message received from mme-app, bytesRead %d", bytesRead);
 			insert_job(g_tpool_tipc_reader_s11, handle_mmeapp_message_s11, tmpBuf);
 		}
-		memset(buffer, 0 , 1024);
+		memset(buffer, 0 , S11_IPC_MSG_BUF_LEN);
```

Comment:
for performance reason this is not something we should do... TIPC header should have length field. We achieve nothing by setting memory to 0. Please get this out.

Reply:
Can you clarify this ? We use the same buffer for different messages hence memset-ing each time.

Reply:
Does your tipc messasge has header ?? What we achieve by resetting the memory to 0???
tipc message header should have length and it tells how much data to access from buffer.

Reply:
Thanks for clarifying. Removed this line.